sematic/ui/packages/common/src/typeViz/views/hugging_face_stored_model.tsx

```diff
@@ -0,0 +1,27 @@
+import { ValueComponentProps, ViewComponentProps} from "src/typeViz/common";
+
+export function HuggingFaceStoredModelShortView(props: ValueComponentProps) {
+    return HuggingFaceStoredModelView(props, true);
+}
+
+export function HuggingFaceStoredModelFullView(props: ViewComponentProps) {
+    return HuggingFaceStoredModelView(props, false);
+}
+
+function HuggingFaceStoredModelView(props: ValueComponentProps | ViewComponentProps, short: boolean) {
+    const { valueSummary } = props;
+    const { values } = valueSummary;
+    const modelType = values.peft_model_type ? values.peft_model_type : values.model_type;
+    const modelTypePieces = modelType.split(".");
+    const modelTypeShortName = modelTypePieces[modelTypePieces.length - 1];
```

Comment:
This is fine, but as a tip: we also have the lodash [last](https://lodash.com/docs/4.17.15#last) available to use.